Web Experiences/Blueprints/alert-bar/alertBar.css

```diff
@@ -52,29 +65,43 @@ body.show-TopBanner {
     line-height: 10px;
 }
 
-#bx_TopBanner .bx_TopBanner__p--span {
-    color: [[Title Text Colour | colour|#4D4D4D|{order: 4}]];
-    font-family: [[Title Font | enum(Arial,Arial Narrow,Brush Script MT,Calibri,Cambria,Candara,Copperplate,Courier,Courier New,Didot,Garamond,Geneva,Helvetica,Lucida Bright,Monaco,Optima,Perpetua,Times,Times New Roman,Verdana) |Calibri|{order: 5}]];
-    font-size: [[ Title Text Font Size | number | 18 | {order: 4;}]]px;
+/* text */
+#bx-{{ref}} #bx_TopBanner .bx_TopBanner__p--span {
+    color: [[Title Text Colour | colour |#444| { group: Text Configuration }]];
+    font-size: [[ Title Text Font Size | number | 18 | { group: Text Configuration }]]px;
     line-height: 16.1px;
     font-weight: 500;
     padding-right: 40px;
 }
 
-#bx_TopBanner .bx_TopBanner__p--button {
+/* CTA button */
+#bx-{{ref}} #bx_TopBanner .bx_TopBanner__p--button {
     border-radius: 4px;
     color: #fff;
-    background-color: [[Button Colour | colour |#2196f3| {order: 9}]];
-    border-color: [[Button Line Colour | colour |#2196f3| {order: 10}]];
-    font-family: [[Button Text Font | enum(Arial,Arial Narrow,Brush Script MT,Calibri,Cambria,Candara,Copperplate,Courier,Courier New,Didot,Garamond,Geneva,Helvetica,Lucida Bright,Monaco,Optima,Perpetua,Times,Times New Roman,Verdana) |Calibri|{order: 10}]];
     padding: 10px 23px;
-    margin-right: 30px;
+    display: inline-block;
+    min-width: 110px;
+    text-align: center;
     cursor: pointer;
+    border: none;
+    margin-right: 10px;
+    color: [[ Button Text Color | colour | #fff | { required: true, group: Button Configuration } ]];
+    background-color: [[ Button Color | colour | #4caf50|{ required: true, group: Button Configuration } ]];
```

Comment:
Do we have a pattern/policy for what fields are required and not?

Reply:
Links and images only is usually my way to go, thank you that you pointed this out